packages/dd-trace/src/config.js

```diff
@@ -452,35 +376,14 @@ class Config {
   }
 
   #applyStableConfig (config, obj) {
-    const {
-      DD_APPSEC_ENABLED,
-      DD_APPSEC_SCA_ENABLED,
-      DD_DATA_STREAMS_ENABLED,
-      DD_DYNAMIC_INSTRUMENTATION_ENABLED,
-      DD_ENV,
-      DD_IAST_ENABLED,
-      DD_LOGS_INJECTION,
-      DD_PROFILING_ENABLED,
-      DD_RUNTIME_METRICS_ENABLED,
-      DD_SERVICE,
-      DD_VERSION
-    } = config
-
-    this.#setBoolean(obj, 'appsec.enabled', DD_APPSEC_ENABLED)
-    this.#setBoolean(obj, 'appsec.sca.enabled', DD_APPSEC_SCA_ENABLED)
-    this.#setBoolean(obj, 'dsmEnabled', DD_DATA_STREAMS_ENABLED)
-    this.#setBoolean(obj, 'dynamicInstrumentation.enabled', DD_DYNAMIC_INSTRUMENTATION_ENABLED)
-    this.#setString(obj, 'env', DD_ENV)
-    this.#setBoolean(obj, 'iast.enabled', DD_IAST_ENABLED)
-    this.#setBoolean(obj, 'logInjection', DD_LOGS_INJECTION)
-    const profilingEnabled = normalizeProfilingEnabledValue(DD_PROFILING_ENABLED)
-    this.#setString(obj, 'profiling.enabled', profilingEnabled)
-    this.#setBoolean(obj, 'runtimeMetrics.enabled', DD_RUNTIME_METRICS_ENABLED)
-    this.#setString(obj, 'service', DD_SERVICE)
-    this.#setString(obj, 'version', DD_VERSION)
+    this.#applyConfigValues(config, obj, {})
```

Comment:
The third argument means we will fill that object but never use it, as far as I can tell. Should we potentially just not set the unprocessed values, if we do not pass through a third argument?